packages/perseus/src/widgets/radio/choice.jsx

```diff
@@ -456,97 +232,155 @@ class Choice extends React.Component<$FlowFixMe, State> {
-
-        // We only show cross-out in certain products, and
-        // only when you're still _answering_ the question. We also show this
-        // for the SAT product area.
-        const crossOutEnabled = this.props.apiOptions.crossOutEnabled;
-        const showCrossOutMenu = crossOutEnabled && !this.props.showCorrectness;
-
         // We want to show the choices as dimmed out when the choices are
         // disabled. However, we don't want to do this in the SAT product and
         // we also don't want to do this when we're in review mode in the
         // content library.
         const showDimmed =
             (!sat && !reviewMode && this.props.apiOptions.readOnly) ||
-            (showCrossOutMenu && this.props.crossedOut);
+            this.props.crossedOut;
 
         return (
-            <div className={sat ? satRadioMenuContainer : undefined}>
-                <CrossOutMenuWrapper
-                    enabled={showCrossOutMenu}
-                    pos={this.props.pos}
-                    primaryProductColor={
-                        this.props.apiOptions.styling.primaryProductColor
-                    }
-                    crossedOut={this.props.crossedOut}
-                    onCrossedOutChange={this._setCrossedOut}
-                    isSatProduct={sat}
+            <div
+                style={{
+                    dispay: "flex",
+                    flexDirection: "column",
+                }}
+            >
+                <div
+                    style={{
+                        display: "flex",
+                        flexDirection: "row",
+                        width: "100%",
+                        opacity: showDimmed ? 0.5 : 1.0,
+                    }}
                 >
-                    <LabelOrDiv
-                        htmlFor={
-                            !this.props.editMode
-                                ? commonInputProps.id
-                                : undefined
+                    <Clickable
+                        onClick={() => {
+                            this.props.onChange({
+                                checked: true,
+                                crossedOut: false,
+                            });
+                        }}
+                        className={descriptionClassName}
+                        disabled={
+                            this.props.disabled ||
+                            this.props.apiOptions.staticRender ||
+                            this.props.apiOptions.readOnly
                         }
-                        className={className}
-                        style={{opacity: showDimmed ? 0.5 : 1.0}}
+                        aria-label={`Select Choice ${getChoiceLetter(
+                            this.props.pos,
+                        )}`}
+                        aria-selected={this.props.checked}
+                        style={{
+                            flex: 1,
+                        }}
```

Comment:
**Suggestion (blocking):** Given we were originally rendering an `input` with a specific role ("checkbox" or "radio"), I think we should specify that role on the `Clickable`.  This sets the `role` attribute on the element it renders and customizes the event handlers to replicate the behaviour of the different aria roles.  Different roles will activate in response to different keyboard events, e.g. space bar vs enter.

Reply:
Thanks for adding `role` to this `Clickable`.